docs/topics/signals.rst

```diff
@@ -307,6 +307,33 @@ spider_error
     :param spider: the spider which raised the exception
     :type spider: :class:`~scrapy.Spider` object
 
+feed_slot_closed
+~~~~~~~~~~~~~~~~
+
+.. signal:: feed_slot_closed
+.. function:: feed_slot_closed(slot)
+
+    Sent when a :ref:`feed exports <topics-feed-exports>` slot is closed.
+
+    This signal supports returning deferreds from its handlers.
+
+    :param slot: the slot closed
+    :type failure: scrapy.extensions.feedexport._FeedSlot
```

Comment:
```suggestion
    :type slot: scrapy.extensions.feedexport._FeedSlot
```

Reply:
It also looks wrong that the public API now exposes an underscore-prefixed class :)

Reply:
Hmm, that’s a good point. Although addressing that properly would not be trivial, i.e. we should probably review the _FeedSlot API and decide which parts to keep public and which parts to make private as we rename _FeedSlot to FeedSlot.

On the bright side, no need for a deprecation period for _FeedSlot, at [it does not seem overused](https://www.google.com/search?q=%22_FeedSlot%22&nfpr=1). But it would be trivial to add one, I think we have functions or decorators for that, so it might be still worth considering.

Reply:
It makes sense to either make `FeedSlot` class public or to create an equivalent class to inform users when a feed has been exporter somewhere, the number of items, size  and other attributes...

Reply:
`_FeedSlot` is a class that takes 9 arguments (file, exporter, storage, uri, format, store_empty, batch_id, uri_template, filter), and stores them as public attrbiutes and also stores `itemcount` (modified externally) and `_exporting` (modified internally). We probably don't need to expose all of these, though maybe we do.

Reply:
We have decided to make `_FeedSlot` public and to keep the old private name with a deprecation warning.

Reply:
@guillermo-bondonno  Could you please do this?

Reply:
Sure